### PR TITLE
Fix more type warnings.

### PR DESCRIPTION
**Pull request by hysw**

### CODE REVIEW

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 1]
```diff
@@ -937,7 +937,8 @@ void GraphicsBenchmarkApp::ProcessKnobs()
             SetupSpheres();
         }
         else {
-            const bool requireMoreSpheres = sphereInstanceCountKnobChanged && (pSphereInstanceCount->GetValue() > mInitializedSpheres);
+            const uint32_t initializedCount   = static_cast<uint32_t>(pSphereInstanceCount->GetValue());
+            const bool     requireMoreSpheres = sphereInstanceCountKnobChanged && (initializedCount > mInitializedSpheres);
```
Comment (apazylbe) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:941]: I wonder, do we need to check `sphereInstanceCountKnobChanged`?

**Review by apazylbe — APPROVED** [review]
One question, although not directly related to this PR, otherwise lgtm

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 1 above)
Comment (hysw) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:941]: sphereInstanceCountKnobChanged is not the best name for it. It is "user has released the slider and values are final". I'll add this weirdness to the knob design update doc.

Comment (apazylbe) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:941]: ah, cool, thanks

**Review by elviscapiaq — APPROVED** [review]

**Pull request merged**